src/util/Xhr.js

```diff
@@ -376,6 +383,10 @@ class Xhr {
      * @return {void}
      */
     abort(): void {
+        if (this.axios && this.axiosSource) {
+            this.axiosSource.cancel('Operation canceled due to new request.');
```

Comment:
Double check if this does console logging.
Since we don't really want it to log.

Also don't think the `if` check does anything since it will always be true as they are never set to null.

Reply:
👍 that was mostly for my testing! Removing it